JoonasHeinonen/escape-from-zordoom-prison
# Feature/improved menu

### [Create a background for the level select](https://trello.com/c/kEFACMKh/72-create-a-background-for-the-level-select)

https://user-images.githubusercontent.com/33086486/236057688-41df45aa-8af8-464a-961d-9986e570463a.mp4

**Changes:**
 - Imported more tiles, images and improved the sprite sheets for both Rivet and Angela.
 - Added space background for the main menu.
 - Added the proper scene for the level selection: Prison cell in which you can see the emotion of each character.
 - Added all the main menu functionalities, sans level selection to their separate scene.
 - Improved the player UI so that it detects the input from the enter key.
 - Removed unused scene to optimise the games resources.

**How to test:**
 - See that the space looks good for the main menu.
 - See that the Angela and Rivet smiling/being disappointed at your choice of the player character.
 - Verify that the enter works properly on each of the UIs and that it accepts input from the keyboard.

**Sources cited:**
 - [Access a sibling node](https://godotengine.org/qa/36018/access-a-sibling-node)
